content/blog/2019-09-11-migrate-from-helm-v2-to-helm-v3.md

```diff
@@ -213,4 +215,65 @@ $ helm3 2to3 convert postgres --tiller-out-cluster
 
 Very cool and simple, right :-)
 
+## Clean up of Helm v2 data
+
+The last step is cleaning up the old data. While this is not required, we strongly recommend it.
+
+Let's check available options:
+
+```
+$ helm3 2to3 cleanup -h
+cleanup Helm v2 configuration, release data and Tiller deployment
+
+Usage:
+  2to3 cleanup [flags]
+
+Flags:
+      --dry-run                  simulate a command
+  -h, --help                     help for cleanup
+  -l, --label string             label to select tiller resources by (default "OWNER=TILLER")
+  -s, --release-storage string   v2 release storage type/object. It can be 'secrets' or 'configmaps'. This is only used with the 'tiller-out-cluster' flag (default "secrets")
+  -t, --tiller-ns string         namespace of Tiller (default "kube-system")
+      --tiller-out-cluster       when  Tiller is not running in the cluster e.g. Tillerless
+```
+
+It will clean:
+- Configuration (Helm home directory)
+- v2 release data
+- Tiller deployment
+
+And of course the safest way is to start with `--dry-run` flag:
+
+```
+$ helm3 2to3 cleanup --dry-run
+NOTE: This is in dry-run mode, the following actions will not be executed.
+Run without --dry-run to take the actions described below:
+
+WARNING: Helm v2 Configuration, Release Data and Tiller Deployment will be removed.
+This will clean up all releases managed by Helm v2. It will not be possible to restore them if you haven't made a backup of the releases.
+Helm v2 will not be usable afterwards.
+
+[Cleanup/confirm] Are you sure you want to cleanup Helm v2 data? [y/N]: y
+
+Helm v2 data will be cleaned up.
+[Helm 2] Releases will be deleted.
+[Helm 2] ReleaseVersion "redis.v1" will be deleted.
+[Helm 2] Tiller service in "kube-system" namespace will be removed.
+[Helm 2] Home folder "/Users/rimas/.helm" will be deleted.
+```
+
+It will show what releases going to be deleted, Tiller service to be removed from `kube-system` namespace and Helm v2 home folder will be deleted.
+
+When you are ready to clean up Hem v2 data, just run that command without `--dry-run` flag.
+
+**NOTE:** The `cleanup` command will remove the Helm v2 Configuration, Release Data and Tiller Deployment. It cleans up all releases managed by Helm v2. It will not be possible to restore them if you haven't made a backup of the releases. Helm v2 will not be usable afterwards.
+
+If you are using Tillerless Helm v2, just add `--tiller-out-cluster` to clean up Helm v2 data.
+
+The plugin also supports non default Helm v2 `home` data folder and Tiller releases `namespace`:
+
+```
```

Comment:
Add : $ export HELM_V2_HOME=$PWD/.helm2

Reply:
done already in https://github.com/helm/helm-www/pull/285/commits/f84d5c422066ff9f1e960d31c92bc2637d76b4a4